src/standalone/api/kernels/index.ts

```diff
@@ -29,18 +31,6 @@ export function getKernelsApi(extensionId: string): Kernels {
                 });
                 return;
             }
-            const execution = kernelProvider.getKernelExecution(kernel);
-            if (!isRemoteConnection(kernel.kernelConnectionMetadata) && execution.executionCount === 0) {
-                // For local kernels, execution count must be greater than 0,
-                // As we pre-warms kernels (i.e. we start kernels even though the user may not have executed any code).
-                // The only way to determine whether users executed code is to look at the execution count
-                sendTelemetryEvent(Telemetry.NewJupyterKernelsApiUsage, undefined, {
-                    extensionId,
-                    pemUsed: 'getKernel',
-                    accessAllowed
-                });
-                return;
```

Comment:
Not sure why this telemetry point was removed

Reply:
this one was removed since it was in the if block I removed but maybe we'll still need it based on comment above

Reply:
Please can you bring in the logic to check if `public async start(options?: IDisplayOptions):` was invoked with `disableUI = false`
E.g. keep a property on kernel to indicate how it was started, then check that variable here..

E.g. create a property 
```typescript
public get userStartedKernel() {
    return this.startupUI.disableUI;
}
```

This way, we still have a way of not returning kernels until and unless it was explicitly started by users (which is what we hoped for in the earlier condition).

Reply:
added!